jmx/src/main/java/io/airlift/jmx/CacheStatsMBean.java

```diff
@@ -28,7 +28,7 @@ public CacheStatsMBean(Cache<?, ?> cache)
     }
 
     @Managed
-    public long size()
+    public long getSize()
```

Comment:
This not being exported looks more like a bug in code which processes @Managed properties.

cc: @martint @electrum